build-tests/x86/tumbleweed/test-image-nitro-enclave/appliance.kiwi

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<image schemaversion="7.5" name="kiwi-test-image-nitro-enclave">
+    <description type="system">
+        <author>Marcus Schäfer</author>
+        <contact>marcus.schaefer@suse.com</contact>
+        <specification>AWS Nitro Enclave test build</specification>
+    </description>
+    <profiles>
+        <profile name="default" description="CPIO: default profile" import="true"/>
+        <profile name="std" description="KERNEL: default kernel" import="true"/>
+    </profiles>
+    <preferences>
+        <version>1.1.1</version>
+        <packagemanager>zypper</packagemanager>
+        <rpm-excludedocs>true</rpm-excludedocs>
+        <rpm-check-signatures>false</rpm-check-signatures>
+    </preferences>
+    <preferences>
+        <type image="enclave" enclave_format="aws-nitro" kernelcmdline="some"/>
```

Comment:
aws-nitro can be a bit misleading. Some words on AWS terminology:

**Nitro -** The current generation virtualization stack. Nitro is what runs EC2 instances.
**Nitro Enclaves -** EC2 Technology that allows an instance to take some of its resources and spawn a new instance from that. Connection to parent is limited to vsock.
**Nitro Enclave -** A sibling instance that an EC2 instance created based on its own resources.

So - albeit a bit long - I think it makes sense to call this "aws-nitro-enclaves". Or alternatively "eif", because the output file of the process is an .eif file.

Reply:
ok then let's name this ```enclave_format="eif"```

Reply:
done